src/lib/server/Tibber.ts

```
@@ -0,0 +1,365 @@
+
+	// The new structure for everything!
+	status = {
+		home: structuredClone(statusInitValues),
+		cabin: structuredClone(statusInitValues)
+	};
+
+	// Singleton
+	private static __instance: Tibber;
+
+	public static getInstance() {
+		if (!this.__instance) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)